app/models/user.rb

```diff
@@ -5,14 +5,26 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  ## Validations
+  validates_presence_of :role, if: :other_roles
+  validate :check_role, if: :other_roles
+
   ## Associations
   belongs_to :company
   has_many :invoices, through: :company
   has_many :clients, through: :company
 
+  ## Constants
+  ROLE = [["Auditor","auditor"], ["Accountant","accountant"], ["Manager","manager"]]
```

Comment:
`ROLE` should be `ROLES`, an array of roles in downcase. To use roles as option to select box create constant `ROLES_OPTIONS` in respective helper module and use User::ROLES to give values to ROLES_OPTIONS`